hilti/toolchain/src/compiler/validator.cc

```diff
@@ -281,13 +305,10 @@ struct VisitorPost : visitor::PreOrder, public validator::VisitorMixIn {
 
         if ( auto attrs = n->attributes() )
             for ( const auto& attr : attrs->attributes() ) {
-                if ( attr->kind() != hilti::Attribute::Kind::RequiresTypeFeature )
-                    error(fmt("invalid attribute '%s' for function parameter", attr->attributeName()), n);
-
-                else {
+                // Already validated all attributes are RequiresTypeFeature
```

Comment:
The comment seems misleading. Isn't it more like "already flagged attributes other than RequiresTypeFeature as errors"?

Reply:
I'll just remove it since it's just assumed with the new model anyway, but you're right it was misleading